private-templates-service/client/src/store/currentTemplate/reducers.ts

```diff
@@ -67,15 +73,34 @@ export function currentTemplateReducer(state = initialState, action: CurrentTemp
         templateJSON: action.templateJSON,
         templateName: action.templateName,
         sampleDataJSON: action.sampleDataJSON,
+        version: action.version,
```

Comment:
same thing as above, I don't believe `action.version` gets updated in the action `receiveExistingTemplateUpdate`. This one might be trickier to solve as you need the backend to tell you whether the version is the same or something new. perhaps this is a future task

Reply:
Thank you for the very thorough review!!!